tests/pr_test.go

```diff
@@ -12,12 +12,19 @@ import (
 const resourceGroup = "geretain-test-key-protect"
 const terraformDir = "examples/default"
 
+var ignoreUpdates = []string{
+	"module.key_protect_module.ibm_kms_instance_policies.key_protect_instance_policies",
```

Comment:
Is this just a once off for this PR? If so, please leave a code comment here stating that it should be removed once this PR is merged